joblib/_dask.py

```diff
@@ -339,6 +344,9 @@ async def f(func, callback):
 
         return cf_future
 
+    def retrieve_result_callback(self, out):
+        return _retrieve_wrapped_call(out)
```

Comment:
Or maybe this is not possible because of limitations to test coverage measurements when running with the dask backend?

Reply:
Actually since https://github.com/joblib/joblib/pull/1520/files#diff-850d691284c501e195917aad188c687dfbfcd33f6c9f963d58f094d7aa0c5db0R1300 is not covered either, I suspect we don't collect the coverage data with distributed. Maybe because it's too slow?

I think we should try to enable coverage for dask.

Reply:
Any use of the dask backend will result in this line being executed, it is be covered. Looking into activating coverage reporting for dask.

Reply:
Coverage seems broken, there are those logs in all pipelines:

![image](https://github.com/joblib/joblib/assets/29153872/8b9de188-fffd-4ce8-872a-a8cb67f0a47f)

Reply:
Otherwise there is no difference between distributed and non distributed pipelines, coverage configuration looks identical.

Reply:
I have fixed the error messages, but it doesn't fix the lack of coverage report in the inline diff on github. It looks like coverage is in fact good but there is a desync issue between github and codecov. Investigation.

Reply:
Wouldn't the proper way be to only upload once to codecov, when all the pipelines are over and after appending all reports in a pipeline that gather from all other test pipelines ?

Reply:
Resolving conversation following https://github.com/joblib/joblib/pull/1520#issuecomment-1805496225